File: ex00/my_string_index.py
"""
:type  param_1: {String}
:type  param_2: {Character}
:rtype: integer
"""
def my_string_index(param_1, param_2):
    for i in range(len(param_1)):
        if param_1[i] == param_2:
            return i
    
    return 0

File: js-quest06/ex00/my_string_index.js
/*
**
** QWASAR.IO -- my_string_index
**
**
** @param {String} param_1
** @param {Character} param_2
** @return {integer}

**
*/


function my_string_index(haystack, needle) {
	var index = 0;

	while (index < haystack.length) {
		if (haystack[index] == needle) {
			return index;
		}

		index++;
	}

	return index;
};
